File: HLIOSPublisher/HLIOSPublisher/HLIOSPublisher/ViewController.m
//
//  ViewController.m
//  HLIOSPublisher
//
//  Created by 星宇陈 on 14/7/7.
//  Copyright (c) 2014年 Emiaostein. All rights reserved.
//

#import "ViewController.h"
#import "appMakerSDK/appMaker.h"

static NSString* const demoBookDirName = @"book";

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
}

- (void)viewDidAppear:(BOOL)animated {
    
    NSString *demoBookPath = [[[NSBundle mainBundle] bundlePath] stringByAppendingPathComponent:demoBookDirName];
    
    [appMaker openBookWithRootViewController:self
                          bookDirectoryPath:demoBookPath
                                theDelegate:(id<appMakerDelegate>)self
                             hiddenBackIcon:YES
                            hiddenShareIcon:YES];
}

@end

File: SACPublisher/SACPublisher/ViewController.swift
//
//  ViewController.swift
//  SACPublisher
//
//  Created by Emiaostein on 21/02/2017.
//  Copyright © 2017 botai. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    override func viewDidAppear(_ animated: Bool) {
        super.viewDidAppear(animated)
        setup()
    }
    
    private func setup() {
        let bookDir = Bundle.main.bundleURL.appendingPathComponent("book", isDirectory: true)
        appMaker.openBook(withRootViewController: self, bookDirectoryPath: bookDir.path, theDelegate: nil, hiddenBackIcon: true, hiddenShareIcon: true)
    }
}

